orchagent/portsorch.cpp

```diff
@@ -1400,8 +1451,11 @@ bool PortsOrch::setPortAdminStatus(Port &port, bool state)
     attr.id = SAI_PORT_ATTR_ADMIN_STATE;
     attr.value.booldata = state;
 
+    // if sync between cmis module configuration and asic is supported,
+    // do not change host_tx_ready value in STATE DB when admin status is changed.
+
     /* Update the host_tx_ready to false before setting admin_state, when admin state is false */
-    if (!state)
+    if (!state && !m_cmisModuleAsicSyncSupported)
```

Comment:
@noaOrMlnx can we use the newly introduced API setHostTxReady() here as well instead of directly calling hset?

Reply:
Done